dask/array/core.py

```diff
@@ -4720,4 +4729,19 @@ def from_npy_stack(dirname, mmap_mode="r"):
     return Array(dsk, name, chunks, dtype)
 
 
+def new_da_object(dsk, name, chunks, meta=None, dtype=None):
+    """Generic constructor for dask.dataframe objects.
```

Comment:
```suggestion
    """Generic constructor for dask.array or dask.dataframe objects.
```

Possible copy-paste leftover.

It seems like we can't have a `new_?_object`, since this requires a `chunks` argument, while `new_dd_object` doesn't.